Automation/azKeyVault/deployment.ps1

```diff
@@ -0,0 +1,67 @@
+<#
+ .SYNOPSIS
+    Deploys a template to Azure
+ .DESCRIPTION
+    Deploys an Azure Resource Manager template
+ .PARAMETER subscriptionId
+    The subscription id where the template will be deployed.
+ .PARAMETER resourceGroupName
+    The resource group where the template will be deployed. Can be the name of an existing or a new resource group.
+ .PARAMETER templateFilePath
+    Optional, path to the template file. Defaults to template.json.
+ .PARAMETER parametersFilePath
```

Comment:
This should not be a file. Parameters should be passed to the ARM template in the form of ParameterObject